### ISSUE TITLE
Preparing Vina Sans for Google Fonts

### ISSUE THREAD

**Issue by yanone** [nguyentype/vinasans#2]
Hello,

I'm tasked with preparing Vina Sans for release on Google Fonts. I checked the font to comply with GF specifications and found the most glaring problem to be the glyph set. The font is missing many many glyphs that GF requires as the minimum.

Many glyphs are composites of accented letters, those should be easy to add. Others may have to be drawn. Please take a look at [this file](https://github.com/googlefonts/glyphsets/blob/main/GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist) and download and place it in your sources folder next to the .glyphs file. When you open Glyphs.app, you will see new Custom Filters in the left sidebar that will help you adding them. Your font must support the "GF_Latin_Core" glyph set as the minimum.

Otherwise there are a small handful of tiny issues that can be corrected easily and I can help you with that once the glyph set is complete.

Please do ask if you have any questions.
Thank you.

**Comment by nguyentype** [comment]
Hello @yanone,

Thank you for your help. I just updated the font files and source files into the repository.
These files included a new style for the font: NGT Vina Sans Neue (The old version will be NGT Vina Sans Original now)

**Comment by yanone** [comment]
I'll take a look next week. Thank you.

**Comment by yanone** [comment]
What is, in your opinion, the difference between the two masters? They look fairly identical in most cases. Am I correct in thinking that only one of the two would get released?

**Comment by nguyentype** [comment]
Hi @yanone , The difference between the masters is: 
- Different glyphs (like I, G,...)
- Different marks (acute, grave,...)
You can try to type out some glyphs with Vietnamese letters to see the differences. Here is a pangram: 
Trường ở quê sạch và đẹp lắm do bố xây kĩ

**Comment by yanone** [comment]
I can see the differences. But what do we make of them? Do you want to release both versions on Google Fonts even though the differences are not very pronounced?

**Comment by nguyentype** [comment]
Hi @yanone, after considering, I think I will release Vina Sans Neue only. And The font name will be renamed to Vina Sans instead of Vina Sans Neue.

**Comment by yanone** [comment]
Great. I'll be off next week. You'll hear from me again the week after.

**Comment by yanone** [comment]
I'm starting my actual work on the font now. When I'm done, I will create a Pull Request (PR) to send my changes back to you, which you will be asked to merge into your repository.
Meanwhile, please fill [this form](https://docs.google.com/forms/d/e/1FAIpQLSeMwHN8J213ZaxHrr5lHCrX56HY_NjGrWB8o604g98YxuMrdA/viewform) with the bio to be displayed next to the font on fonts.google.com.
Thank you.

**Comment by nguyentype** [comment]
@yanone Thank you 😁

**Comment by yanone** [comment]
Done. Please merge this: https://github.com/nguyentype/vinasans/pull/3
and don't forget to fill the designer profile form (link in comment above).
Thank you.

**Comment by nguyentype** [comment]
@yanone I merged & filled the form. Thank you 🙏
